integration-tests/emulated/chains/parachains/people/people-kusama/src/genesis.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Substrate
+use sp_core::storage::Storage;
+
+// Cumulus
+use cumulus_primitives_core::ParaId;
+use emulated_integration_tests_common::{build_genesis_storage, collators, SAFE_XCM_VERSION};
+use parachains_common::Balance;
+
+pub const PARA_ID: u32 = 1004;
+pub const ED: Balance = people_kusama_runtime::ExistentialDeposit::get();
+
+pub fn genesis() -> Storage {
```

Comment:
I'm not sure, @NachoPal ?

Reply:
In general it should be possible to do: `RuntimeGenesis::default().build()` to populate externality storage with the genesis values.  
But as i see we have this copy&pasted in all runtimes, so maybe can keep this for a future follow-up to clean them all.

Reply:
There's some refactoring and improvement from @michalkucharczyk in polkadot-sdk on genesis config, maybe he has an opinion on how this can be done best with minimal copy+pasta.

Reply:
I am not very familiar with this part of code. But if I understand correctly this function is later used for some integration tests. So it is not strictly runtime function, just a testing helper?

Reply:
When it comes to duplication: If the set of customized fields in genesis config is the same for every runtime/test then we could use single _json patch_ and use GenesisBuilder runtime API to build the storage. But this approach would also require that serialization format is exactly the same for every runtime (e.g. SessionKeys).

However json-patch approach may slow down execution of the tests (this concern was raised by @NachoPal if I recall correctly) - as we need to execute a runtime call to get the state.